src/app/components/pages/settings/outputs/outputs.component.ts

```diff
@@ -13,6 +13,7 @@ import { QrCodeComponent } from '../../../layout/qr-code/qr-code.component';
 })
 export class OutputsComponent implements OnInit {
 
+  isLoading = false;
```

Comment:
This is not really necessary, checking `!wallets` is enough, similar to how it is done in `HistoryComponent`.

Reply:
Fixed.